audio/music/jingleBellsKuba.mp3.license

```diff
@@ -0,0 +1,9 @@
+Sound by Kuba Michalski
+
+# Attribution 4.0 International (CC BY 4.0)
+By using this file you accept that it won't be used for destructive purposes.
```

Comment:
This additional line is problematic as I do not know what destructive means. This is in the license:

> No warranties are given. The license may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material.

Please see if this covers your concerns.

**I can not guarantee when I hear your music, that no-one else uses it for destructive purposes.**

Reply:
Idea: please remove this line.